cob_twist_controller/src/inverse_jacobian_calculations/inverse_jacobian_calculation.cpp

```diff
@@ -143,17 +144,17 @@ Eigen::MatrixXd PInvDirect::calculate(const TwistControllerParams& params,
         ROS_ERROR("PInvDirect does not support SVD. Use PInvBySVD class instead!");
     }
 
-    double lambda = db->getDampingFactor(Eigen::VectorXd::Zero(1, 1), jacobian);
+    Eigen::MatrixXd lambda = db->getDampingFactor(Eigen::VectorXd::Zero(1, 1), jacobian);
     if (cols >= rows)
     {
         Eigen::MatrixXd ident = Eigen::MatrixXd::Identity(rows, rows);
-        Eigen::MatrixXd temp = jacobian * jac_t + lambda * lambda * ident;
+        Eigen::MatrixXd temp = jacobian * jac_t + lambda(1, 1) * lambda(1, 1) * ident;
         result = jac_t * temp.inverse();
     }
     else
     {
         Eigen::MatrixXd ident = Eigen::MatrixXd::Identity(cols, cols);
-        Eigen::MatrixXd temp = jac_t * jacobian + lambda * lambda * ident;
+        Eigen::MatrixXd temp = jac_t * jacobian + lambda(1, 1) * lambda(1, 1) * ident;
```

Comment:
please fix this as suggested in my [comment](https://github.com/ipa320/cob_control/pull/124/files#r95815791)